ri4:
# Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

# Position Controllers ---------------------------------------
  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint1
    pid: {p: 100.0, i: 0.01, d: 10.0}

# Position Controllers ---------------------------------------
  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint2
    pid: {p: 100.0, i: 0.01, d: 10.0}

# Position Controllers ---------------------------------------
  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint3
    pid: {p: 100.0, i: 0.01, d: 10.0}

# Position Controllers ---------------------------------------
  joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint4
    pid: {p: 100.0, i: 0.01, d: 10.0}

# Position Controllers ---------------------------------------
  joint5_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint5
    pid: {p: 100.0, i: 0.01, d: 10.0}

# Position Controllers ---------------------------------------
  joint6_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint6
    pid: {p: 100.0, i: 0.01, d: 10.0}

# Position Controllers ---------------------------------------
  joint7_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint7
    pid: {p: 100.0, i: 0.01, d: 10.0}

# Position Controllers ---------------------------------------
  joint8_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint8
    pid: {p: 100.0, i: 0.01, d: 10.0}

# Position Controllers ---------------------------------------
  joint9_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint9
    pid: {p: 100.0, i: 0.01, d: 10.0}

# Position Controllers ---------------------------------------
  joint10_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint10
    pid: {p: 100.0, i: 0.01, d: 10.0}
